CHANGELOG.md

```diff
@@ -3,19 +3,22 @@ x.x.x Release notes (yyyy-MM-dd)
 
 **This project is in the Alpha stage. All API's might change without warning and no guarantees are given about stability. Do not use it in production.**
 
+### Features
```

Comment:
[nit] I'd typically prefer to have features in the enhancements section as the difference between a feature and an enhancement is quite blurry.

Reply:
I think that Flexible Sync is a new for Dart SDK. And it is the first time we are introducing it in this CHANGELOG. That's why I added it under Features. But I can change it.

Reply:
Done